lib/rubocop/cop/style/space_inside_string_interpolation.rb

```diff
@@ -5,22 +5,36 @@ module Cop
     module Style
       # This cop checks for whitespace within string interpolations.
       #
-      # Good:
-      #    var = "This is the #{good} example"
+      # @example
+      #   Good:
```

Comment:
You can convert good and bad into line comments, for consistency with the rest of our code.